documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -976,13 +970,256 @@ the spin-wave centre:
    centre at :math:`[0,-1,1]`.
 
 
+.. _cylindrical_projection:
+
 Cylindrical Projections
```

Comment:
While I am not repeating them here, all the comments in the spherical section apply here and should be addressed.